consensus/istanbul/backend/announce.go

```diff
@@ -170,25 +170,31 @@ func (sb *Backend) announceThread() {
 				}
 			}
 
-			updateAnnounceVersionFunc()
-			go sb.generateAndGossipAnnounce(announceVersion)
-
 			// Use this timer to also prune all announce related data structures.
 			if err := sb.pruneAnnounceDataStructures(); err != nil {
 				logger.Warn("Error in pruning announce data structures", "err", err)
 			}
 
-		case <-announceVersionsCheckTicker.C:
```

Comment:
The signedAnnounceVersions will still need to be intermittently checked with its peers.